endoreg_db/urls.py

```
@@ -1,75 +1,83 @@
-from .views.raw_pdf_meta_validation_views import UpdateSensitiveMetaView
-from .views.raw_pdf_anony_text_validation_views import RawPdfAnonyTextView, UpdateAnonymizedTextView
-from .views.examination_views import (
-    ExaminationViewSet,
-    VideoExaminationViewSet,  # NEW: Add the VideoExaminationViewSet
+from .views.video.segmentation import VideoViewSet, VideoLabelView, UpdateLabelSegmentsView
+from .views.video.timeline import video_timeline_view
+from .views.pdf.raw_pdf_meta_validation_views import PDFFileForMetaView
+from .views.pdf.raw_pdf_meta_validation_views import UpdateSensitiveMetaView
+from .views.pdf.raw_pdf_anony_text_validation_views import RawPdfAnonyTextView, UpdateAnonymizedTextView
+from .views.patient_examination import (
     get_morphology_classification_choices_for_exam,
```

Ruff (0.12.2)

`.views.patient_examination.get_morphology_classification_choices_for_exam` imported but unused

Remove unused import

(F401)